### ISSUE TITLE
Dynamic import: unused expression

### ISSUE THREAD

**Issue by screendriver** [angelozerr/tslint-language-service#45]
- __TSLint version__: 5.5.0
- __TSLint language service__: 0.9.6
- __TypeScript version__: 2.4.1

#### TypeScript code being linted

```ts
if (process.env.NODE_ENV !== 'production') {
  import('my-dev-package');
}
```

with `tslint.json` configuration:

```json
{
  "extends": ["tslint:recommended", "tslint-react", "tslint-config-prettier"]
}
```

#### Actual behavior

In Visual Studio Code I get following error: `[tslint] unused expression, expected an assignment or function call (no-unused-expression)`

Strangely if I run `tslint` from the command line I don't get this error  

#### Expected behavior

No error should be shown.

**Comment by screendriver** [comment]
Nevermind. I forgot to change the TypeScript language level in my VSCode workspace to `2.4.1` 🙄
